### PR TITLE
Hides experience level from the EventDetailActivity

### PR DESCRIPTION
**Pull request by tiwiz**
As Francesco requested, the experience level is not there anymore... It's actually only set to GONE in the code :-1:

### CODE REVIEW

File: app/src/main/java/com/ls/ui/activity/EventDetailsActivity.java [hunk 1]
```diff
@@ -227,30 +227,9 @@ private void fillDate(@NonNull EventDetailsEvent event) {
         }
     }
 
-    private void fillPreDescription(@NonNull EventDetailsEvent event) {
+    private void fillPreDescription() {
         LinearLayout layoutPreDescription = (LinearLayout) findViewById(R.id.layoutPreDescription);
```
Comment (rock3r) [inline app/src/main/java/com/ls/ui/activity/EventDetailsActivity.java]: Can't we delete it too?

Comment (rock3r) [inline app/src/main/java/com/ls/ui/activity/EventDetailsActivity.java]: I mean the view in this layout thing

Comment (tiwiz) [inline app/src/main/java/com/ls/ui/activity/EventDetailsActivity.java]: Yes, I'll do that right now

**Comment by tiwiz** [comment]
@rock3r I deleted the layout as suggested, sorry I did not do it properly the first time :-)

**Comment by rock3r** [comment]
LGTM let's #shipit

**Pull request merged**